indico/modules/events/timetable/templates/display/balloons/block.html

```diff
@@ -0,0 +1,88 @@
+{% from 'attachments/_management_info_column.html' import render_attachment_info %}
+
+<div class="balloon">
+    <div class="title-container session-block">
+        <div class="title">
+            {{- block.session.title }}
+            {%- if block.title -%}
+                <span class="balloon-subtitle"> ({{ block.title }})</span>
+            {%- endif -%}
+        </div>
+    </div>
+    <div class="description">{{ block.session.description }}</div>
+    <span class="hr"></span>
+    <div class="section-title-container">
+        <div class="title-header">
+            <div class="title-caption">{% trans %}Block{% endtrans %}</div>
+            <div class="group right entry-action-buttons">
+                <a class="icon-edit i-button-icon js-dialog-action push-balloon-back js-edit"
+                   title="{% trans %}Edit block{% endtrans %}"
+                   data-title="{% trans title=block.title %}Edit block '{{ title }}'{% endtrans %}"></a>
+                <a href="#" class="icon-remove i-button-icon push-balloon-back js-delete"
+                   title="{% trans %}Delete timetable entry{% endtrans %}"
+                   data-title="{% trans title=block.title %}Delete timetable entry '{{ title }}'?{% endtrans %}"
+                   data-confirm="{% trans title=block.title %}Are you sure you want to delete timetable entry '{{ title }}'?{% endtrans %}"></a>
```

Comment:
and since this deletes the whole block i'd use "session block" instead of "timetable entry" in the title/prompt